package/Dockerfile.submariner-buildenv

```diff
@@ -0,0 +1,43 @@
+FROM fedora:33
+ARG TARGETPLATFORM
+
+# dnf cache and base packages, shared across all build sub-directories
+RUN dnf -y --setopt=install_weak_deps=0 --nodocs \
+           --installroot /output/base --releasever 33 \
+           install glibc bash glibc-minimal-langpack coreutils-single
```

Comment:
This seems very boiler-platey, how about adding a simple script that gets copied in before the run and calling it?

e.g.
```suggestion
# dnf cache and base packages, shared across all build sub-directories
RUN INSTALL_ROOT=base KEEP_CACHE=y BASE_RUN=y \
    /dnf_install.sh glibc bash glibc-minimal-langpack coreutils-single
```

with `dnf_install.sh` being something like:
```bash
#!/usr/bin/env bash
[[ -n $BASE_RUN ]] || cp -a /output/base /output/${INSTALL_ROOT}
dnf -y --setopt=install_weak_deps=0 --nodocs \
    --installroot /output/${INSTALL_ROOT} --releasever ${FEDORA_VERSION} \
    install ${*}
[[ -n $KEEP_CACHE ]] || dnf -y --installroot /output/${INSTALL_ROOT} --releasever  ${FEDORA_VERSION} clean all
```